src/shared/lib/StoreUtils.ts

```diff
@@ -118,13 +118,13 @@ export async function fetchClinicalData(clinicalDataMultiStudyFilter:ClinicalDat
 }
 
 export async function fetchClinicalDataInStudy(studyId:string,
-                                               clinicalDataSingleStudyFilter:ClinicalDataSingleStudyFilter,
+                                               clinicalDataSingleStudyFilter:ClinicalDataSingleStudyFilter, clinicalDataType: 'SAMPLE' | 'PATIENT',
```

Comment:
maybe add the new parameter in a new line (to be consistent with the rest of the parameters)